test/unit/math/rev/functor/ad_stack_matrix_test.cpp

```diff
@@ -0,0 +1,75 @@
+#include <stan/math.hpp>
```

Comment:
Missing resize tests. Also missing some sorta test for `AD_allocator`

Reply:
Added tests